pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -1077,6 +1081,31 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		return nil, errors.Wrap(err, "failed to ensure Certificate")
 	}
 
+	log.Print("Checking OVN Octavia driver support")
+	octaviaProviderSupport, err := IsOctaviaVersionSupported(client, MinOctaviaVersionWithProviders)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to determine if Octavia supports providers")
+	}
+
+	octaviaProvider := "default"
+	if octaviaProviderSupport {
+		page, err := providers.List(lbClient).AllPages()
+		if err != nil {
+			log.Print("failed to get lbs provider list, using default octavia provider")
+		} else {
+			providers, err := providers.ExtractProviders(page)
+			if err != nil {
+				return nil, errors.Wrap(err, "failed to extract providers list")
+			}
+			for _, provider := range providers {
+				if provider.Name == OVNProvider {
```

Comment:
Is this a standardized name or can admin name providers however he likes? I mean - is it like flavors?

Reply:
That is static, and seems it must be 'ovn' for the ovn-octavia driver